client/pages/my-garage.jsx

```diff
@@ -20,6 +23,62 @@ class MyCars extends React.Component {
     });
   }
 
+  deleteCar(event) {
+    if (event) {
+      event.preventDefault();
+    }
+    const { cars, vehicleIndex } = this.state;
+    fetch(`/api/garage/delete-car/${cars[vehicleIndex].vehicleId}`, {
+      method: 'DELETE'
+    })
+      .then(result => result.json())
+      .then(result => {
+        const carsCopy = [...cars];
+        carsCopy.splice(vehicleIndex, 1);
+        this.setState({
+          cars: carsCopy,
+          deleteModal: false
+        });
+      })
+      .catch(err => console.error(err));
+  }
+
+  toggleDeleteModal(event, index) {
+    if (event) {
+      event.preventDefault();
+    }
```

Comment:
If you don't actually can about the event object and you don't actually need to prevent any default behavior, please refactor your code to not take `event` as a parameter.